packages/wp-source/src/libraries/api.ts

```diff
@@ -46,7 +46,7 @@ class Api {
 
   async getIdBySlug(endpoint: string, slug: string) {
     const response = await this.get({ endpoint, params: { slug } });
-    const [entity] = await response.json();
+    const [entity] = await response.clone().json();
```

Comment:
Why do you have to clone the response?

Reply:
I don't fully understand the problem, but it seems that you cannot  call `json()` twice for the same `Response` object (at least it happens with [`node-fetch`](https://github.com/bitinn/node-fetch/issues/533#issuecomment-429261523)). 😕

I think it would be a good idea to change the return type of `api.get()` in the next version. At first I found useful to work directly with a `Response` object but now I'm not very convinced.

Reply:
I'm afraid `clone` can be really slow on a big response. We should find an alternative.

Reply:
Or take a look and see how expensive it really is.

Reply:
Anyway, I've been able to get rid of `response.clone` calls.